src/components/ContactUsForm/index.jsx

```diff
@@ -0,0 +1,120 @@
+/*
+ * ContactUsForm consists of validation and elements of contact form, imports Netlifyform for post data.
+ * The base code that is used in both netlifyForm and index are implemented from https://github.com/jon-fm/react-ssg-netlify-forms-demo
+ */
+import React, { useState } from "react";
+import { useTranslation } from "react-i18next";
+import NetlifyForm from "../NetlifyForm";
+import { Col, Row } from "antd";
+import { RiSendPlaneFill } from "react-icons/ri";
+import "./index.css";
+
+const ContactUsForm = () => {
+  const [t] = useTranslation();
+  const [submissionMessage, setSubmissionMessage] = useState("");
+  // beforeSubmit is for validations and disabling submit button
+  const [processing, setProcessing] = useState(false);
+  const beforeSubmit = async () => {
+    if (
+      contactFormValues.name.length > 0 &&
+      contactFormValues.email.length > 0
+    ) {
+      setProcessing(true);
+      // Wait 2 seconds to simulate async delay
+      await new Promise((resolve) => setTimeout(resolve, 2000));
+      return true;
+    } else {
+      return false;
+    }
+  };
```

Comment:
comment in line 15 should be moved to line 16

Reply:
is `processing `used to create spinner ? You should clarify what it does

Reply:
Does `submissionMessage` display something on screen when form is sent? You should clarify

Reply:
in line 19 and 20, instead of using a magic number you should use a const. `MIN_USER_INPUT_LENGTH` maybe? I'm not good at naming things.

Reply:
Thanks Tarkan, I'll make the changes that you've mentioned.